package.json

```diff
@@ -100,7 +100,7 @@
     "build:commonjs": "BABEL_ENV=cjs babel ./src --extensions \".js,.jsx,.ts,.tsx\" -d lib --copy-files",
     "build:esm": "BABEL_ENV=esm babel src -d esm --extensions \".js,.jsx,.ts,.tsx\" --copy-files",
     "build:umd": "webpack",
-    "build:pack": "npm run clean && npm run build:commonjs && npm run tsc:commonjs && npm pack --pack-destination './examples'",
+    "build:pack": "npm run clean && npm run build:commonjs && npm run build:esm && npm run tsc:commonjs && npm run tsc:esm && npm pack --pack-destination './examples'",
```

Comment:
esm was omitted to keep build times low.  One of the previous merges changed this package.json to have main go out of esm/index.js instead of lib/index.js.  I forgot the update the build:pack.

We should change this to also remove the build:commonjs and tsc:commonjs to keep those times low